packages/presentation/lib/src/bloc/game_bloc.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:domain/domain.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:presentation/src/bloc/state.dart';
+import 'package:presentation/src/bloc/events.dart';
+
+const maxAttempts = 3;
+const initIsGuessTrue = false;
+const suggestedNumberInit = -1;
+
```

Comment:
Why u just can't make it final inside bloc? Or make separate file for constant? I see this almost in every file.

Reply:
it will be smth like this?
class A{
final int maxAttempts = 3;
final int attemtsCounter = maxAttempts;
}
on my constant is better for reusability and and readability